source/assets/stylesheets/base/_variables.scss

```diff
@@ -12,9 +13,13 @@ $heading-line-height: 1.2;
 
 // Other Sizes
 $base-border-radius: 3px;
+$large-border-radius: 24px;
 $base-spacing: $base-line-height * 1em;
+$medium-spacing: $base-spacing * 1.25;
 $small-spacing: $base-spacing / 2;
+$xsmall-spacing: $base-spacing / 4;
 $base-z-index: 0;
+$min-height-overflow-freeze: 750px;
```

Comment:
Do you wanna split these into different sections? I know it's a small site, so it probably doesn't matter, but could be border radius, spacing, then leave the z-index and height in other.